app/resources/numismatic_accessions/numismatic_accession_change_set.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class NumismaticAccessionChangeSet < Valkyrie::ChangeSet
+  delegate :human_readable_type, to: :model
+
+  property :date, multiple: false, required: false
+  property :person, multiple: false, required: false
+  property :firm, multiple: false, required: false
+  property :accession_number, multiple: false, required: false
+  property :type, multiple: false, required: false
+  property :cost, multiple: false, required: false
+  property :account, multiple: false, required: false
+  property :note, multiple: false, required: false
+  property :private_note, multiple: false, required: false
+
+  property :member_ids, multiple: true, required: false, type: Types::Strict::Array.of(Valkyrie::Types::ID)
+
+  validates_with AutoIncrementValidator, property: :accession_number
+  validates_with MemberValidator
+
+  def primary_terms
+    [
+      :date,
+      :person,
+      :firm,
+      :accession_number,
```

Comment:
We should probably not let users edit `accession_number` since we're assigning an auto-incremented value.